cumulusci/tasks/apex/testrunner.py

```diff
@@ -308,11 +315,6 @@ def _run_task(self):
             url=self.tooling.base_url + "runTestsAsynchronous",
             json={"classids": ",".join(str(id) for id in ids)},
         )
-        if result.status_code != httplib.OK:
```

Comment:
por que?

Reply:
`_call_salesforce` already takes care of checking the status and raising SalesforceGeneralError